packages/create-cloudflare/src/workers.ts

```diff
@@ -52,7 +52,7 @@ export async function addWorkersTypesToTsConfig(ctx: C3Context) {
 	const typesEntrypoint = `@cloudflare/workers-types/${entrypointVersion}`;
 
 	try {
-		const config = jsonc.parse(tsconfig);
+		const config = jsonc.parse(tsconfig, [], { allowTrailingComma: true });
```

Comment:
Is this just an unrelated bugfix?

Reply:
yes, I think this is a valid change but I do think that I should not include it here

I think I just added it when I was hitting issues with trailing comma issues before, but after I removed all the create-cloudflare toml->jsonc changes so this should not be needed in this PR anymore, I'll remove this here and see if CI is happy, I'll also reintroduce this in a followup PR if that works for you 🙂

Reply:
I've been looking into this and maybe this change is not as valid as I thought it was. The C3 wrangler files are not part of the monorepo's formatting: https://github.com/cloudflare/workers-sdk/blob/d34ef3d5b0e45470e6c06a7bd24a498f85b98af8/.prettierignore#L16-L19 so they don't necessarily need to include trailing commas, including them in the monorepo formatting doesn't seem really necessary, so for the time being I think that it's fine keeping the code as is and update this later when/if needed 🙂